src/org/stringtemplate/v4/Interpreter.java

```diff
@@ -1393,7 +1393,7 @@ protected void trace(InstanceScope scope, int ip) {
     protected void printForTrace(StringBuilder tr, InstanceScope scope, Object o) {
         if ( o instanceof ST ) {
             if ( ((ST)o).impl ==null ) tr.append("bad-template()");
-            else tr.append(" "+((ST)o).impl.name+"()");
+            else tr.append(" ").append(((ST) o).impl.name).append("()");
```

Comment:
📝 This code base typically omits a space after a cast.

Reply:
Good catch! Must have happened when applying the fixit suggestion with IntelliJ.

```suggestion
            else tr.append(" ").append(((ST)o).impl.name).append("()");
```